imx.mk

```diff
@@ -37,8 +37,8 @@ BOOT_IMG		?= $(ROOT)/out/boot.img
 ################################################################################
 # Targets
 ################################################################################
-all: tfa u-boot linux optee-os buildroot flash-image
-clean: tfa-clean buildroot-clean u-boot-clean optee-os-clean
+all: optee-os tfa u-boot linux buildroot flash-image
```

Comment:
The target order doesn't matter from a functional standpoint. If at least you were following some kind of convention (alphabetical order...) it could make sense, but as it is, not really :wink: 
I suggest to not touch this line.